repos.d/aosc.yaml

```diff
@@ -0,0 +1,21 @@
+###########################################################################
+# AOSC OS
+###########################################################################
+- name: aosc
+  type: repository
+  desc: AOSC OS Packages
```

Comment:
Is "OS" part really necessary? I don't see it widely used on e.g. https://packages.aosc.io/, and it adds a bit of visual noise, while providing no additional information.

Reply:
@AMDmi3 Yes, I do apologise if our website is not explicit enough about this information. "AOSC" refers to our community name "Anthon Open Source Community", while "AOSC OS" refers to our actual Linux distribution project.